test/e2e/network/netpol/kubemanager.go

```diff
@@ -181,12 +171,35 @@ func (k *kubeManager) probeConnectivity(args *probeConnectivityArgs) (bool, stri
 	}
 
 	commandDebugString := fmt.Sprintf("kubectl exec %s -c %s -n %s -- %s", args.podFrom, args.containerFrom, args.nsFrom, strings.Join(cmd, " "))
-	stdout, stderr, err := k.executeRemoteCommand(args.nsFrom, args.podFrom, args.containerFrom, cmd)
-	if err != nil {
-		framework.Logf("%s/%s -> %s: error when running command: err - %v /// stdout - %s /// stderr - %s", args.nsFrom, args.podFrom, args.addrTo, err, stdout, stderr)
-		return false, commandDebugString, nil
```

Comment:
this will be much simpler if you use existing methods, it is a very small change this way

```
err := wait.PollImmediate(3*time.Second, time.Minute, func() (bool, error) {
stdout, stderr, err := k.executeRemoteCommand(args.nsFrom, args.podFrom, args.containerFrom, cmd)
	if err != nil {
		framework.Logf("Retrying ... %s/%s -> %s: error when running command: err - %v /// stdout - %s /// stderr - %s", args.nsFrom, args.podFrom, args.addrTo, err, stdout, stderr)
		return false, nil
        }
        return true, nil
}
if err != nil {
return false, commandDebugString, nil
}
return true, commandDebugString, nil
```

Reply:
thank you for your feedback @aojea. I have added this now.